common/src/config.rs

```diff
@@ -233,13 +233,16 @@ struct Http {
     timeout: u32,
     /// Maximum interval between failed request retries in seconds.
     max_retry_interval: u32,
+    /// Array of certificates to additionally trust when sending HTTPS requests.
+    tls_trust_roots: Vec<PathBuf>,
```

Comment:
Opinions on where this option should go? I feel like all of the TLS config is quite a bit of a mess right now. Let's talk f2f